File: src/main/java/guru/xgm/language/wts/psi/tree/WtsIToken.java
package guru.xgm.language.wts.psi.tree;

import com.intellij.psi.tree.IElementType;
import guru.xgm.language.wts.lang.WtsLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class WtsIToken extends IElementType {

    public WtsIToken(@NotNull @NonNls String debugName) {
        super(debugName, WtsLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "WtsIToken." + super.toString();
    }
}

File: src/main/kotlin/raft/war/language/wts/psi/tree/WtsIToken.kt
package raft.war.language.wts.psi.tree

import com.intellij.psi.tree.IElementType
import raft.war.language.wts.lang.WtsLanguage
import org.jetbrains.annotations.NonNls

class WtsIToken(debugName: @NonNls String) : IElementType(debugName, WtsLanguage.instance) {
    override fun toString(): String {
        return "WtsIToken." + super.toString()
    }
}
